src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -25,4 +25,38 @@ public void equals_False_X10() {
     Integer i = Integer.valueOf(10);
     assertNotEquals(rnX, i);
   }
+
+  @Test
+  public void equals_True_Reflexivity() {
+    assertEquals(rnX, rnX);
+  }
+
+  @Test
+  public void equals_True_Symmetric() {
+    RomanNumeral rn2 = new RomanNumeral("X");
+    assertEquals(rnX.equals(rn2), rn2.equals(rnX));
+  }
+
+
+  @Test
+  public void equals_True_Transitive() {
+    RomanNumeral rn2 = new RomanNumeral("X");
+    RomanNumeral rn3 = new RomanNumeral("X");
+
+    assertEquals(rnX.equals(rn2) && rn2.equals(rn3), rnX.equals(rn3));
+  }
+
+  @Test
+  public void equals_True_Consistent() {
```

Comment:
Unit tests aren't useful for consistency. I appreciate your trying to be comprehensive, but you should remove this test.